README.md

```diff
@@ -11,6 +11,8 @@ _RoboMaker sample applications include third-party software licensed under open-
 
 - [ROS Kinetic](http://wiki.ros.org/kinetic/Installation/Ubuntu) / [ROS Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu) - Other versions may work, however they have not been tested
 - [Colcon](https://colcon.readthedocs.io/en/released/user/installation.html) - Used for building and bundling the application.
+- [vcs](https://github.com/dirk-thomas/vcstool#how-to-install-vcstool) - Used to pull in sample app dependencies
```

Comment:
I think this should say `vcstool` (software name) instead of `vcs` (executable name).

Reply:
I would add "sample app dependencies that are only available from source, not from `apt` or `pip`".